source/mobile/mobile-overview.rst

```diff
@@ -1,37 +1,84 @@
+|                                                                |                                                                     |  
+| - Easiest way to deploy Mattermost Apps.                       | You maintain full control over the distribution of applications,    |
+| - Test push notifications using the Test Push                  | including the look, feel, and capabilities of your mobile app.      |
+|   Notification Service (TPNS).                                 |                                                                     |
+| - Mattermost Enterprise includes a                             | **Limitations:**                                                    |
+|   Hosted Push Notification Service (HPNS).                     |                                                                     |
+| - Apps update automatically with the latest                    | - Requires development knowledge and resources to maintain mobile   |
+|   features, enhancements, and security updates.                |   app code as Mattermost releases new product updates.              |
+|                                                                | - Must deploy your own push proxy server.                           |
+| **Limitations:**                                               |                                                                     |
+|                                                                |                                                                     |  
+| - Can’t white-label Mattermost Mobile Apps.                    |                                                                     |
+| - Can’t deploy your own push proxy server.                     |                                                                     |  
++----------------------------------------------------------------+---------------------------------------------------------------------+
 
-.. note::
-    App wrapping via an EMM provider is untested, and not supported by Mattermost. Instead, we recommend using an EMM provider that supports Managed App Configuration.  
+Technical and Security Requirements
+-----------------------------------
+
+See our `Supported Devices/Mobile Device Requirements <https://docs.mattermost.com/install/requirements.html#mobile-apps>`__ documentation for basic mobile device requirements, and our `Supported Mattermost Server Versions <https://docs.mattermost.com/administration/mobile-changelog.html>`__ documentation for details on Mattermost Server minimum requirements. 
+
+We recommend running the latest version of the Mattermost Server as it contains the most recent features and applicable security updates. If this isn't possible, we encourage you to be on the most recent Extended Support Release version of Mattermost. This release has critical feature updates that will ensure compatibility in a number of areas, including the Mattermost Push Notification Service (MPNS).
```

Comment:
I would add to keep the MPNS also updated to the latest version if they are hosting their own